PC/_wmimodule.cpp

```diff
@@ -243,6 +246,21 @@ _wmi_exec_query_impl(PyObject *module, PyObject *query)
         }
     }
 
+    // If current user doesn't have permission to query the WMI, the function
+    // IWbemLocator::ConnectServer will hang for 5 seconds, and there is no way
+    // to specify the timeout. So we use an Event object to simulate a timeout.
+    // More detail: gh-112278.
```

Comment:
```suggestion
    // gh-112278: If current user doesn't have permission to query the WMI, the
    // function IWbemLocator::ConnectServer will hang for 5 seconds, and there
    // is no way to specify the timeout. So we use an Event object to simulate
    // a timeout.
```

This is our usual style for these kinds of comments.